mmocr/models/textrecog/backbones/resnet_abi.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+from mmcv.runner import BaseModule, Sequential
+
+import mmocr.utils as utils
+from mmocr.models.builder import BACKBONES
+from mmocr.models.textrecog.layers import BasicBlock
+
+
+@BACKBONES.register_module()
+class ResNetABI(BaseModule):
+    """Implement ResNet backbone for text recognition, modified from
+      `ResNet <https://arxiv.org/pdf/1512.03385.pdf>`_ and
+      `<https://github.com/FangShancheng/ABINet>`_
+    Args:
+        base_channels (int): Number of channels of input image tensor.
+        layers (list[int]): List of BasicBlock number for each stage.
+        channels (list[int]): List of out_channels of Conv2d layer.
+        out_indices (None | Sequence[int]): Indices of output stages.
+        last_stage_pool (bool): If True, add `MaxPool2d` layer to last stage.
+    """
+
+    def __init__(self,
+                 base_channels=3,
+                 arch_settings=[3, 4, 6, 6, 3],
+                 channels=[32, 32, 64, 128, 256, 512],
+                 strides=[1, 2, 1, 2, 1, 1],
+                 out_indices=None,
+                 last_stage_pool=False,
+                 init_cfg=[
+                     dict(type='Xavier', layer='Conv2d'),
+                     dict(type='Constant', val=1, layer='BatchNorm2d')
+                 ]):
```

Comment:
Recommend to keep pace with mmcv/mmdet'ResNet
```suggestion
                 base_channels=3,
                 stem_channels=None,
                 base_channel=32,
                 num_stages=5,
                 arch_settings=[3, 4, 6, 6, 3],
                 channels=[32, 64, 128, 256, 512],
                 strides=[1, 2, 1, 2, 1, 1],
                 out_indices=None,
                 last_stage_pool=False,
                 init_cfg=[
                     dict(type='Xavier', layer='Conv2d'),
                     dict(type='Constant', val=1, layer='BatchNorm2d')
                 ]):  
```
it's weird that channels parameter has different dim with other